realm/realm-library/src/objectServer/java/io/realm/PermissionManager.java

```diff
@@ -151,10 +162,9 @@ public boolean isGlobalRealm() {
     private RealmResults<PermissionOffer> offers;
 
     /**
-     * FIXME Javadoc
      * Creates a PermissionManager for the given user.
      *
-     * Implementation notes: This class is thread safe since all public methods have thread confined checks in them
+     * This class is thread safe since all public methods have thread confined checks in them
```

Comment:
We should consider avoiding the use of "thread safe" in this context. While completely accurate, it seems to be confusing. I did it too.

Maybe something along the lines of "this class is thread confined, so thread safety is not a concern"?  I don't really know...